frontend/src/pages/spotnet/dashboard/withdraw-all/WithdrawAll.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import BalanceCards from 'components/BalanceCards';
+import { ReactComponent as ETH } from '../../../../assets/icons/ethereum.svg';
+import { ReactComponent as USDC } from '../../../../assets/icons/borrow_usdc.svg';
+import { ReactComponent as STRK } from '../../../../assets/icons/strk.svg';
+import './withdraw_all.css';
+import { useWalletStore } from 'stores/useWalletStore';
+import Button from 'components/ui/Button/Button';
+import { ReactComponent as AlertHexagon } from 'assets/icons/alert_hexagon.svg';
+import { axiosInstance } from 'utils/axios';
+
+const WithdrawAll = () => {
+  const { walletId } = useWalletStore();
+  const [balances, setBalances] = useState([
+    { icon: <ETH />, title: 'ETH', balance: '0.00' },
+    { icon: <USDC />, title: 'USDC', balance: '0.00' },
+    { icon: <STRK />, title: 'STRK', balance: '0.00' },
+  ]);
```

Comment:
Can you please move this inside of `BalanceCards` component (here and in `Form.jsx` as I see no reasons to have it outside a component as it is used only inside.

Reply:
Updated!

Reply:
Could you please anyway change it, just remoe from Form.jsx, Stake.jsx and WilthdrawAll.tsx and use this versions with 3 default balances inside `BalanceCards`, thanks

Reply:
@1nonlypiece